### ISSUE TITLE
Error using IIFE AsBind

### ISSUE THREAD

**Issue by tyronedougherty** [torch2424/as-bind#53]
Hi there,

I've run into the following error whenever I try to use the `AsBindIIFE.AsBind.instantiate` function:

```
as-bind.iife.js:11 Uncaught (in promise) Error: Required Exported AssemblyScript Runtime functions are not present. Runtime must be set to "full" or "stub"
```

Any idea what the issue is? I've tried setting the runtime like so in my `package.json`:

```
    "asbuild:untouched": "asc ./node_modules/as-bind/lib/assembly/as-bind.ts assembly/index.ts --target debug --runtime full",
```

But that seems to have had no effect.

**Comment by torch2424** [comment]
Hello! Thank you for the issue! :smile: 

Hmmm I think this issue is coming from the [AssemblyScript loader](https://www.assemblyscript.org/loader.html#loader).

cc @dcodeIO or @MaxGraey , do either of you know why this might be? 

---

In the meantime, does compiling your AssemblyScript module without `--target debug` work? :thinking:

**Comment by MaxGraey** [comment]
I ussume you should update how loader imported according this: https://github.com/AssemblyScript/assemblyscript/pull/1513

**Comment by torch2424** [comment]
Ah! It broke under me, cool I can fix this tomorrow, along with #50 :smile: 

Thanks @MaxGraey ! :+1:

**Comment by tyronedougherty** [comment]
Thanks @torch2424 😄

**Comment by torch2424** [comment]
Fixed and published in 0.5.0 :smile:
